packages/itmat-job-executor/src/setupProxy.js

```diff
@@ -0,0 +1,9 @@
+const { createProxyMiddleware } = require('http-proxy-middleware');
+
+module.exports = function (app) {
+    app.use(createProxyMiddleware('/graphql', {
+        ws: true,
+        target: 'http://localhost:3003',
+        changeOrigin: true
+    }));
+};
```

Comment:
why does job-executor even have graphql endpoint?

Reply:
also what is this commit doing in a PR for mfa?

Reply:
I haven't wrote this code. As @fguitton created the branch for me, I think he will know about this :o

Reply:
As mentioned in my first comment (https://github.com/dsi-icl/itmat-broker/pull/484#issuecomment-622957655), you should rebase your changes ontop of the develop branch which @sou-chon has now fixed regarding the `react-script`/`lerna` incompatibility issue.

Reply:
@fguitton so should I check out the develop branch and add the mfa function from the beginning? or any way better to modify the code in accordance with the develop branch? Thanks